src/crystal/event_loop.cr

```diff
@@ -1,28 +1,41 @@
 require "./event"
 
-module Crystal::EventLoop
-  @@eb = Crystal::Event::Base.new
-  @@dns_base : Crystal::Event::DnsBase?
+class Thread
+  # :nodoc:
+  getter(event_base) { Crystal::Event::Base.new }
 
-  def self.after_fork
-    @@eb.reinit
+  # :nodoc:
+  getter(loop_fiber) do
+    Fiber.new(name: "Event Loop") do
+      loop do
+        self.event_base.run_once
+        Crystal::Scheduler.reschedule
+      end
+    end
   end
+end
+
+module Crystal::EventLoop
+  {% unless flag?(:preview_mt) %}
+    def self.after_fork
+      Thread.current.event_base.reinit
+    end
+  {% end %}
 
   def self.resume
     loop_fiber.resume
   end
 
+  private def self.event_base
+    Thread.current.event_base
+  end
+
   private def self.loop_fiber
```

Comment:
Not used anywhere.

Reply:
I didn't found it in this PR nor with `grep -rn` in master.

Reply:
I'm talking about this very private class method, not the instance method and variable.

Reply:
Sure, but sorry I didn't undestand the code URM you mentioned, where `EventLoop.loop_fiber` is used then?

Reply:
Sure, but sorry I didn't undestand the code URL you mentioned, where `EventLoop.loop_fiber` is used then?

Reply:
Oh ok sorry, thanks. I didn't know this language feature, to call a static method like that.